src/test/resources/blockdevices.pan

```diff
@@ -69,8 +69,15 @@ unique template blockdevices;
             "size", 800,
             "volume_group", "vg0"
             ),
-        )
-	);
+        ),
+    "vxvm", nlist (
+        "vcslab.local", nlist("gnr.0", nlist (
+            "dev_path", "/dev/vx/dsk/vcslab.local/gnr.0",
```

Comment:
Spoke to @ned21 about this as I couldn't find any usage of it in our templates - he explained at least some of these fields are required by Aquilon, so that's why we set them. You're right you can derive everything from the device path, this is actually what we do in templates to automatically create this blockdevice from a filesystem so we don't have to add a disk.

I should submit this type to the template library:

type blockdevices_vxvm_type = {
    "dev_path" : string
    "disk_group" : string
    "volume" : string
    "size" ? long
};

type structure_blockdevices = {
#    include component_type
    "physical_devs" ? blockdevices_disk_type {}
    "volume_groups" ? blockdevices_lvm_type {}
    "logical_volumes" ? blockdevices_logicalvolumes_type {}
    "md" ? blockdevices_md_type {}
    "partitions" ? blockdevices_partition_type {}
    "files" ? blockdevices_file_type {}
    "hwraid" ?  blockdevices_hwraid_type {}
    "vxvm" ? blockdevices_vxvm_type {}
};